modules/cluster-logging-deploy-cli.adoc

```diff
@@ -178,187 +78,38 @@ metadata:
   name: cluster-logging
   namespace: openshift-logging <1>
 spec:
-  channel: "stable" <2>
+  channel: stable <2>
   name: cluster-logging
   source: redhat-operators <3>
   sourceNamespace: openshift-marketplace
 ----
-<1> You must specify the `openshift-logging` namespace.
-<2> Specify `stable`, or `stable-5.<x>` as the channel.
-<3> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the CatalogSource object you created when you configured the Operator Lifecycle Manager (OLM).
+<1> You must specify the `openshift-logging` namespace for logging versions 5.7 and older. For logging 5.8 and later versions, you can use any namespace.
+<2> Specify `stable`, or `stable-x.y` as the channel.
```

Comment:
Leaving formatting as is, since this was preexisting content (mostly moving files around for this PR) and is formatted this way throughout logging docs IIRC